src/pages/Page404/Page404.css

```diff
@@ -0,0 +1,25 @@
+@media all and (min-width: 360px) {
```

Comment:
Smallest phones are 320px. You added min-width: 360px. Everything below 360px will lose styling and text color.
![404-error](https://user-images.githubusercontent.com/38657595/60394629-e63c7200-9b2f-11e9-912f-9434a902ff50.jpg)